src/Employer/Employer.Web/Controllers/VacancyManageController.cs

```diff
@@ -33,7 +33,7 @@ public async Task<IActionResult> ManageVacancy(VacancyRouteModel vrm)
 
             var vacancy = await _orchestrator.GetVacancy(vrm);
 
-            if (vacancy.CanEdit)
```

Comment:
do we need to change the name of this property? This is on employer side so the context comes from that